app/labor/daily_digest_email.rb

```diff
@@ -0,0 +1,27 @@
+class DailyDigestEmail
+  def self.send_daily_digest_email(users = [])
+    new(users).send_daily_digest_email
+  end
+
+  def initialize(users = [])
+    @users = users.empty? ? get_users : users
```

Comment:
Good job of looking at existing functionality as reference for getting this functionality completed.